specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-10-01/examples/Caches_Get.json

```diff
@@ -0,0 +1,94 @@
+{
+  "parameters": {
+    "resourceGroupName": "scgroup",
+    "cacheName": "sc1",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2020-10-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
```

Comment:
>body [](start = 7, length = 4)

You don't have systemData in the examples but it's there in the definition of a Cache resource body.

Reply:
Added.